components/encrypt-video.jsx

```diff
@@ -27,12 +30,14 @@ module.exports = React.createClass({
     this.props.setPageState({
       videoDidStart: true
     });
+    ga.event({category: "Video", action: "Clicked on play button"});
```

Comment:
If they come back asking questions about it I might have to re-deduce the logic, this clearly gets a little messy because it just needed to be thrown together.